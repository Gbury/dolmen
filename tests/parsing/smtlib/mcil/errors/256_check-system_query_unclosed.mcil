(check-system sys :query (q (r) s))