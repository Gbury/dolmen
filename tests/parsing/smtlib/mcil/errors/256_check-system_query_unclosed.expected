File "tests/parsing/smtlib/mcil/errors/256_check-system_query_unclosed.mcil", line 1, character 32-33:
1 | (check-system sys :query (q (r) s))
                                    ^
Error while parsing a check-system query, read the symbol 's',
      but expected a closing parenthesis.
